src/containers/Footer.js

```diff
@@ -6,6 +6,21 @@ import { AppFooter } from '@coreui/react';
 export function Footer(props) {
 
 	let FooterService = props.app.locateService("FooterService");
+	let siteFooterImage = props.app.Config.get("footer_image");
```

Comment:
`props.app.Config.get("footer_image")` is 100% equivalent to `props.footer_image` ... this whole decision block is basically no-op.